src/fonts/load_all.ts

```diff
@@ -16,6 +17,7 @@ export function loadAllMusicFonts(): void {
   loadGonville();
   loadPetaluma();
   loadCustom();
+  loadLeland();
```

Comment:
Adding `loadLeland()` to this `load_all.ts` file means that the main vexflow.js build will get larger, since it now includes Leland. Are we all agreeing to this? I'm fine with making the main vexflow.js larger (we can tag this as the 4.1 release).... but I assume we should have a limit, no? We can't include _every single_ SMuFL font.... :-)

Reply:
Tagging @AaronDavidNewman @0xfe @sschmidTU @mscuthbert ...

I think @AaronDavidNewman said he's fine with making the vexflow.js build larger.

I am also OK with this (for now) as long as we have the individual `vexflow-{single_font_name}.js` variants. Anyone who wants a smaller library can use `vexflow-bravura.js` or `vexflow-gonville.js`, etc...